library/aes.c

```diff
@@ -1045,8 +1045,10 @@ int mbedtls_aes_crypt_cbc( mbedtls_aes_context *ctx,
                     const unsigned char *input,
```

Comment:
Do we want to update the header documentation to indicate errors are propagated upwards from the ECB operation?

Reply:
I don't think it's needed.
1. Our ECB operation returns success, so no error.
2. An alternative implementation will most likely not even call the ECB operation, and call directly the HW crypto engine